scripts/components/OpenSearch/build.sh

```diff
@@ -77,19 +81,54 @@ cp -r ./build/local-test-repo/org/opensearch "${OUTPUT}"/maven/org
 
 [ -z "$PLATFORM" ] && PLATFORM=$(uname -s | awk '{print tolower($0)}')
 [ -z "$ARCHITECTURE" ] && ARCHITECTURE=`uname -m`
+[ -z "$DISTRIBUTION" ] && DISTRIBUTION="None"
 
 case $PLATFORM in
```

Comment:
I would combine and do a `case $PLATFORM-$DISTRIBUTION`, it would make the whole thing not nested and much simpler.

Reply:
I am so surprised that I actually didnt even remotely think about that.
Time to reflect on my approach.
Thanks.

Reply:
I raised it to `$PLATFORM-$DISTRIBUTION-$ARCHITECTURE` and it is now even simpler compares to original.